### ISSUE TITLE
About the demo

### ISSUE THREAD

**Issue by shamangary** [Tramac/Fast-SCNN-pytorch#36]
Hello, impressive repo. Do you have the label of the cityscapes? Also do you have a timeline for ADE20k pretrained model release? Thanks

**Comment by Tramac** [comment]
Hi, you can get the label of the cityscapes from [here](https://www.cityscapes-dataset.com/file-handling/?packageID=1), there is no pretrained model for ADE20k.
